test/Pragmas/add_env_path.C

```diff
@@ -7,8 +7,24 @@
 //------------------------------------------------------------------------------
 
 // RUN: mkdir -p %T/subdir && clang -DCLING_EXPORT=%dllexport -shared %S/call_lib.c -o %T/subdir/libtest%shlibext
-// RUN: %setenv ENVVAR_LIB="%T/subdir" && %setenv ENVVAR_INC="%S/subdir"
-// RUN: cat %s | %cling -I %S -Xclang -verify 2>&1 | FileCheck %s
+// RUN: cat %s | %cling -I %S -DENVVAR_LIB="\"%/T/subdir\"" -DENVVAR_INC="\"%/p/subdir\"" -Xclang -verify 2>&1 | FileCheck %s
+
```

Comment:
I believe this is because #ifdef support (InputValidator fixes) has not made it in yet.